src/marvin/ai/text.py

```diff
@@ -343,9 +347,10 @@ def classify(
 
 
 def generate(
-    type_: Optional[type[T]] = None,
+    target: Optional[type[T]] = None,
     instructions: Optional[str] = None,
     n: int = 1,
+    use_cache: bool = True,
```

Comment:
i wonder if the default should be `False`
```suggestion
    use_cache: bool = True,
```
I'm just thinking about the case where you say

class Person:
   # non trivial schema
```
and the rendered generate prompt balloons in a way that you'd only expect if you already read the docs

Reply:
I'd rather put a token cap on it then disable it, because the real problem here is that if you generate 10 of those people and compare it to the result of generating 1 person 10 times, you'll get a much higher quality answer from the first call without the cache

Reply:
a token cap would make sense to me 👍🏼